src/features/deepnav.tsx

```diff
@@ -382,6 +370,7 @@ const endNavigate = () => {
   if (document.body.classList.contains(NAVIGATE_CLASS)) {
     document.body.classList.remove(NAVIGATE_CLASS);
   }
+  window.removeEventListener("keydown", keyDownListener);
```

Comment:
This is additionally poor on my part because the listener is added on `activeDeepNav()` and `toggleFeature()`

So are you saying it is better to have an event listener always on vs only turning it on when needed?

Reply:
It's more a comment that `isNavigating` is already doing this work. If we want to do the smart listener approach you suggest (this is probably better, but harder to get right), then we should prob get rid of `isNavigating` since it will add confusion